monitoring_versioned_docs/version-22.10/monitoring-resources/monitoring-basics/plugin-packs.md

```diff
@@ -7,23 +7,17 @@ title: Plugin packs
 
 ## What is a Plugin Pack?
 
-A Plugin Pack is a downloadable package containing a set of configuration templates that make it fast and easy to
-monitor your IT infrastructure.
+A Plugin Pack is a downloadable package that contains a set of configuration templates that make monitoring your IT infrastructure fast and easy.
 
 Applying a template from a Plugin Pack is the easiest way to monitor a host.
 
-Plugin Packs consist of 2 elements, which are installed separately:
-
-- A plugin that executes the monitoring commands from a poller. Plugins can be
-  installed using the command line interface, or automatically.
-
-- A pack that contains commands, host templates and service templates.
```

Comment:
Il Plugin Pack include anche i discovery command, service discovery template ed eventualmente anche le regole per i Host discovery. Per non complicare aggiungi questa info in una nuova frase (o quello che ritieni meglio per la leggibilità). L'info importante è quella che hai gia messo tu.